src/P2/gc.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <ref.h>
+
+struct SBB {
+    int n;
+    struct CRef ref;
```

Comment:
Remove `struct` keyword, the compiler should infer that CRef is a class as defined in ref.h